lib/macho/load_commands.rb

```diff
@@ -399,6 +480,19 @@ def initialize(view, cmd, cmdsize, name, timestamp, current_version, compatibili
       @current_version = current_version
       @compatibility_version = compatibility_version
     end
+
+    # @param context [MachO::LoadCcommand::SerializationContext] the context
+    # @return [String] the serialized fields of the load command
+    # @api private
+    def serialize(context)
+      format = Utils.specialize_format(FORMAT, context.endianness)
+      lc_str = SIZEOF # a dylib command can only have one dylib
+      cmdsize = Utils.round(SIZEOF + name.to_s.bytesize + 1, context.cmd_round)
+      npad = cmdsize - SIZEOF - name.to_s.bytesize
+      padded_name = name.to_s + "\x00" * npad
```

Comment:
I'm basically fine with this code, but I'd like to see it being pulled out into a utility method instead of being replicated in every `serialize` method that involves an `lc_str`, as I suggested in a previous code comment: https://github.com/Homebrew/ruby-macho/pull/38#discussion_r70019349